src/main/java/org/assertj/core/api/AbstractByteAssert.java

```diff
@@ -179,6 +180,50 @@ public SELF isNotPositive() {
     return myself;
   }
 
+  /**
+   * Verifies that the actual value is even.
+   * <p>
+   * Example:
+   * <pre><code class='java'> // assertions will pass
+   * assertThat((byte) 12).isEven();
+   * assertThat((byte) 46).isEven();
+   *
+   * // assertions will fail
+   * assertThat((byte) 3).isEven();
+   * assertThat((byte) 15).isEven();</code></pre>
+   *
+   * @return this assertion object.
+   * @throws AssertionError if the actual value is {@code null}.
+   * @throws AssertionError if the actual value is not positive.
+   * @since 3.16.0
```

Comment:
By lack of time I could not review it for 3.16.0 so it's 3.17.0 (promise!)

Reply:
Thanks for the review! I will keep working this week.